Dangerfile

```diff
@@ -41,10 +41,6 @@ milestone_checker.check_milestone_due_date(days_before_due: 2)
 # `files: []` forces rubocop to scan all files, not just the ones modified in the PR
 rubocop.lint(files: [], inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
 
-swiftlint.binary_path = './Pods/SwiftLint/swiftlint'
-swiftlint.lint_all_files = true
-swiftlint.lint_files
-
```

Comment:
Any particular reason why we ended up deciding to run SwiftLint outside of Danger? 🤔

Reply:
I've mentioned in more details [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/22408#issuecomment-1896238267), but my main issue was not being able to run SwiftLint on a plain Ubuntu runner (it was silently failing already for a while on Buildkite -- it probably worked only on a Mac), then not being able to run Danger Ruby on containers that could run SwiftLint.

I believe the fast feedback we get from a SwiftLint only step is an advantage, but to be honest I'm not completely sold either way as the SwiftLint via Danger has its own advantages as well, though it would require us to create and maintain a docker image for Ruby + SwiftLint (not a big deal, I guess). There have been a lot of discussions on this topic like paaHJt-3yo-p2 and paaHJt-4l3-p2...